assets/messages.yml

```
@@ -1359,5 +1359,29 @@ messages:
 
         Please don't forget to search for an existing issue matching yours in the appropriate project before raising a new one.
 
+        %quick_links%
+      fillname: []
+  wrong-project-moved:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - realms
+      name: Wrong project (Moved)
+      shortcut: mov
+      message: |-
+        *Thank you for your report!*
+        
+        However, you have created the report for the wrong project, therefore it is being moved to the correct project. That project might require different or additional information, please check whether the report is correct and complete, *revise incorrect information and add missing information*.
```

Should we use violine's wording then? For me that would be fine.
```suggestion
        However, you have created the report for the wrong project, therefore it is being moved to the correct project. As a result of the move, some information might have become lost or incorrect. *Please check whether the report is still correct and complete, and if necessary, edit it accordingly.*
```

---

I think probably.